src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/FeatureToggleServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.professionalapi.service.impl;
+
+import com.launchdarkly.sdk.LDUser;
+import com.launchdarkly.sdk.server.LDClient;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FeatureToggleServiceImpl {
+
+    @Autowired
+    private final LDClient ldClient;
+
+    @Value("${launchdarkly.sdk.environment}")
+    private String environment;
```

Comment:
can you please create FeatureToggleServiceInterface , declare methods and implement the class FeatureToggleServiceImpl

Reply:
This looks fine..will change it for consistency